spec/models/recipes_spec.rb

```diff
@@ -1,12 +1,14 @@
 require 'rails_helper'
+require './spec/support/features/clearance_helpers'
 
 RSpec.describe Recipe do
   it { is_expected.to validate_presence_of(:name) }
 
   describe '#category_names' do
     subject { recipe.category_names }
 
-    let(:recipe) { described_class.create(name: 'Spaghetti') }
+    let(:user) { create(:user) }
+    let(:recipe) { described_class.create(name: 'Spaghetti', user: user) }
```

Comment:
I think I'd rather just use factory bot here and not worry about the user at all.
```suggestion
    let(:recipe) { create(:recipe, name: 'Spaghetti') }
```